src/sentry/suspect_resolutions/resolved_in_active_release.py

```diff
@@ -0,0 +1,34 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models import Deploy, Group, GroupStatus, Project, Release
+
+
+def is_resolved_issue_within_active_release(issue: Group, project: Project) -> bool:
```

Comment:
I think you can probably get rid of the `project: Project` param here since `group.project` is available.